src/main/java/it/gov/pagopa/payment/service/qrcode/QRCodeAuthPaymentServiceImpl.java

```diff
@@ -65,4 +70,14 @@ public AuthPaymentDTO authPayment(String userId, String trxCode) {
     return authPaymentDTO;
   }
 
+  private void sendAuthPaymentNotification(TransactionInProgress trx, AuthPaymentDTO authPaymentDTO) {
+    //todo mettere if (cercare su reward-calc)
+    try {
+      log.info("[SEND_NOTIFICATION] Sending event to Notification");
```

Comment:
when logging, let's put some contextual information, eg the trxId and the userId, specifying in the message that we are sending the authorization message.